source/_components/ptvsd.markdown

```diff
@@ -0,0 +1,118 @@
+---
+layout: page
+title: "PTVSD Debugger (Visual Studio Code)"
+description: "Debugging from Visual Studio Code."
+date: 2019-04-24 09:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: 
+  - Utility
+ha_qa_scale: internal
+ha_release: 0.93
+---
+
+The `ptvsd` component allows you to connect the ptvsd debugger from visual studio code to connect to home assistant.
```

Comment:
perhaps some extra info what you can do with this component, what can you use it for etc.